include/nng/supplemental/nanolib/conf.h

```diff
@@ -315,6 +315,7 @@ struct conf_exchange_node {
 
 	nng_socket       *sock;
 	nng_mtx          *mtx;
+	char             *exchange_url;
```

Comment:
Just realized this.
I prefer to settle multi-stream query in self-defined protocol. instead of  assigning unique URL to each stream.
@RanMaoyi

Reply:
Just thinking about it feels like a huge project, with a lot of decoupling to do, so we can plan it out later.